File: src/main/Java/boj_step2/boj1330.java
import java.io.*;
import java.util.StringTokenizer;

public class boj1330 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        StringTokenizer tokenizer = new StringTokenizer(reader.readLine());
        int a = Integer.parseInt(tokenizer.nextToken());
        int b = Integer.parseInt(tokenizer.nextToken());
        if (a < b) writer.append("<");
        else if (a > b) writer.append(">");
        else writer.append("==");
        writer.close();
    }
}

File: src/main/Kotlin/others/boj1330.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val (a, b) = r.readLine().split(" ").map { it.toInt() }
    val result = if(a > b) ">" else if(a < b) "<" else "=="
    w.write(result)
    w.close()
}